tags/tag-infrastructure/charter.md

```diff
@@ -1 +1,80 @@
-Charter content here
\ No newline at end of file
+# CNCF TAG Infrastructure
+
+## Mission
+
+To define and advance practices, standards, and assessments related to core cloud native infrastructure areas that support and enable scalable, resilient, secure, and performant cloud native systems, applications, and architectures. This supports the CNCF's technical vision by addressing critical problems faced by adopters and contributing to a robust cloud native ecosystem.
+
+## Scope
+
+This TAG covers the following sub-domains and topics within its scope:
+- Data: Data refers to all digital information created, handled, and utilized by applications. Both structured and unstructured forms of data are included. This includes, but is not limited to the following.
+  - User interactions
+  - System logs
+  - Performance metrics
+  - Configuration settings
+  - Permanent records like database entries
+- Storage: Storage used in cloud-native environments include block stores, file systems, object stores, databases, key-value stores, streaming and messaging, and related caching mechanisms. Our focus is on understanding the following fundamental characteristics and relating them to cloud-native use cases.
+  - Availability
+  - Scalability
+  - Performance
+  - Durability
+  - Consistency
+  - Ease-of-use
+  - Cost
+  - Operational complexity
+- Networking: Core connectivity and traffic management in cloud-native systems. This can include:
+  - DNS, load balancing, and service discovery
+  - Service mesh for inter-service communication
+  - Connectivity and traffic management across environments
+  - Policy enforcement for network security and compliance
+  - Observability and monitoring of network behavior
+- Compute: Encompasses kernel and hypervisor-level compute primitives that underpin cloud-native infrastructure (workloads running in Public/Private Cloud, Edge, IoT, Batch, Big Data, AI/ML/LLMs, etc). This can include:
```

Comment:
@GenPage will reach out to workload foundation TAG to clarify the differences with them.

Reply:
There are no concerns from Workload Foundations TAG, so I am going to resolve this thread as-is.